Trimble.Modus.Components/Controls/TMModal/DisplayAlert.xaml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Modus:PopupPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"

```

Comment:
We should not create another component here. We can just reuse the Modal class and create a wrapper around it.

Reply:
Please check and run my commit